web/models.py

```
@@ -1752,6 +1752,65 @@ def display_name(self):
+    date = models.DateTimeField()
+    link = models.URLField(blank=True)
+    location = models.CharField(max_length=255, blank=True)
+    event_type = models.CharField(
+        choices=[("online", "Online"), ("in_person", "In Person")], default="online", max_length=10
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    creator = models.ForeignKey(User, on_delete=models.CASCADE)  # Creator field
+
+    def clean(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean`

Add return type annotation: `None`

(ANN201)